src/libraries/System.IO.FileSystem.AccessControl/ref/System.IO.FileSystem.AccessControl.cs

```diff
@@ -58,18 +77,18 @@ public sealed partial class FileSecurity : System.Security.AccessControl.FileSys
     }
     public sealed partial class FileSystemAccessRule : System.Security.AccessControl.AccessRule
     {
-        public FileSystemAccessRule(System.Security.Principal.IdentityReference identity, System.Security.AccessControl.FileSystemRights fileSystemRights, System.Security.AccessControl.AccessControlType type) : base (default(System.Security.Principal.IdentityReference), default(int), default(bool), default(System.Security.AccessControl.InheritanceFlags), default(System.Security.AccessControl.PropagationFlags), default(System.Security.AccessControl.AccessControlType)) { }
```

Comment:
Are these formatting changes intentional? IIRC, `/t:GenerateReferenceSource` generates extra space after `base`.

Reply:
I did not execute a command using `/t:GenerateReferenceSource`. What's the full command I should run?
I probably just need to revert the change.

Reply:
After building the src and having the src shape that you want you need to run `dotnet msbuild /t:GenerateReferenceSource` in the source project.